GoogleUtilities/ISASwizzler/Public/GoogleUtilities/GULObjectSwizzler.h

```diff
@@ -55,7 +55,7 @@ typedef OBJC_ENUM(uintptr_t, GUL_ASSOCIATION){
  *  @param association The mechanism to use when associating the objects.
  */
 + (void)setAssociatedObject:(id)object
-                        key:(NSString *)key
+                        key:(const void *)key
```

Comment:
Is this a breaking a change?

Reply:
Also, does it make sense to scope things down to `const char *`?

edit: could the `const void *` be why this is non-breaking?

Reply:
It is a breaking change. I used `const void *` just to mirror the parameter in the underlying runtime API method.